src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceObject = {};
+
+  const sourceArray = sourceString
+    .split(';')
+    .map(item => item.split(':').map(el => el.trim()))
+    .filter(el => el.length > 1);
+
+  for (const subArray of sourceArray) {
+    sourceObject[subArray[0]] = subArray[1];
+  }
```

Comment:
You can use array destructuring in the for loop to make the code more concise, easier to read and to avoid unnamed variables like array[0]. It wi Here is how to do it - https://www.freecodecamp.org/news/array-destructuring-in-es6-30e398f21d10/

Reply:
done